### PR TITLE
Correct theme selection assets and function, Painting model and added backbutton function one back in history

### PR DESCRIPTION
**Pull request by kyunwang**
#11 #19 
- Created painting db model
- Added themes images
- Created theme json with images
- Fixed back button
- Theme selection function added

P.S. Need to fix that a theme can only be added once

### CODE REVIEW

File: src/components/Header.vue [hunk 1]
```diff
@@ -18,14 +20,8 @@
...
 		methods: {
-			goBack() {
-				if (this.$route.path !== '/') {
-					console.log('Going back');
-					this.$router.go(-1);
-				}
-			},
 			checkRoute() {
-				console.log(this.$route);
+				// console.log(this.$route);
```
Comment (VincentKempers) [inline src/components/Header.vue]: ??

File: src/router.js [hunk 2]
```diff
@@ -4,6 +4,7 @@ import Router from 'vue-router';
 import Home from './views/Home.vue';
 import Tourmap from './views/Tourmap.vue';
 import Selection from './views/Selection/index.vue';
+// import SelectionOverview from './views/SelectionOverview/index.vue';
```
Comment (VincentKempers) [inline src/router.js]: ??

File: src/views/Selection/index.vue [hunk 3]
```diff
@@ -26,87 +27,186 @@
...
+				selectedThemes: [],
+				isDisabled: true,
 			};
 		},
 		methods: {
-			test() {
-				console.log('Got clicked');
-			},
 			addTheme(theme) {
 				// console.log('Add', evt.target, theme);
```
Comment (VincentKempers) [inline src/views/Selection/index.vue]: delete the comments that you don't need

File: src/views/Selection/index.vue [hunk 4]
```diff
@@ -26,87 +27,186 @@
...
+			content: "";
+			min-height: 1.5rem;
+			max-height: 1.5rem;
+			width: 100%;
+			position: absolute;
+			z-index: -1;
+			bottom: 0;
+			background: #9C0E1C;
+		}
+		// border-bottom: .15rem solid #9C0E1C;
```
Comment (VincentKempers) [inline src/views/Selection/index.vue]: Delete this one too

File: src/views/Selection/index.vue [hunk 5]
```diff
@@ -119,7 +219,7 @@
 	}
 
 	li {
-		border-radius: .3rem;
+		// border-radius: .3rem;
```
Comment (VincentKempers) [inline src/views/Selection/index.vue]: delete this

File: src/router.js (hunk 2 above)
Comment (kyunwang) [reply src/router.js]: This is for the eventual selectionoverview which we will be skipping for now

**Pull request merged**